tests/vm_test.go

```diff
@@ -991,8 +992,11 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			vm := libvmi.NewVirtualMachine(libvmifact.NewAlpine(), opts...)
 			vm.Namespace = testsuite.GetTestNamespace(vm)
 
-			vmJson, err := tests.GenerateVMJson(vm, workDir)
-			Expect(err).ToNot(HaveOccurred(), "Cannot generate VMs manifest")
+			data, err := json.Marshal(vm)
```

Comment:
In my opinion, we should not be in the business of testing `kubectl`. Can you you please delete the tests that use this function instead, e.g in a follow-up pr?

Reply:
Somewhere there are tests that verify the columns printed by `kubectl` look like we are expecting them too. IMO that is a valid test case. But these tests should probably dropped.

See https://github.com/kubevirt/kubevirt/issues/12709

Reply:
(unrelated to this PR)

> Somewhere there are tests that verify the columns printed by kubectl look like we are expecting them too. IMO that is a valid test case

We may want to have an integration test with `kubectl`, but I don't think it makes sense to start a VM (with all the time, race condition and wasted resources) just for that.

> But these tests should probably dropped.

Yes. `GenerateVMJson` was a code smell.